app/views/events/_table.html.erb

```diff
@@ -28,13 +28,16 @@
 
       <td>
 
+      <% if DateTime.now() >= event.datetime %>
```

Comment:
event.datetime might be null

Reply:
Fixed if statement to allow user to attend events with datetime=nil